lib/tune.cpp

```diff
@@ -787,7 +819,8 @@ namespace quda
         param.aux = make_int4(-1, -1, -1, -1);
         tunable.initTuneParam(param);
 
-        while (tuning) {
+        constexpr int candiate_iterations = 2;
```

Comment:
probably want to leave this as a variable that can be changed by the user

Reply:
I'm fine with baking this in at 2, combined with some minimum time requirement.

Reply:
we have this as a virtual function now